lib/logstash/outputs/loggly.rb

```diff
@@ -59,6 +59,12 @@ class LogStash::Outputs::Loggly < LogStash::Outputs::Base
   # https://www.loggly.com/docs/source-groups/
   config :tag, :validate => :string, :default => "logstash"
 
+  # Retry count
+  config :retry_count, :validate => :number, :default => 5
+
+  # Can Retry
```

Comment:
If we keep this setting, then the comment here could use a bit more detail. To inform the user - what behavior does this change?